distribution/docs/src/main/resources/_contents/_data/isr-attributes-table-contents.adoc

```diff
@@ -33,27 +33,27 @@
 |
 
 |isr.original-source
-|A STANAG 4545 ISOURCE.
+|This field contains a description of the source of the data.  Refer to STANAG 4545 ISORCE.
```

Comment:
For definitions that reference external specs, especially when those specs are not easily accessible, can we include a few example values so people have some idea what to expect from these fields?